external/mbedtls/Makefile

```diff
@@ -74,12 +77,12 @@ SRC_CRYPTO_CSRCS =    aes.c           aesni.c         arc4.c          \
                       sha256.c        sha512.c        threading.c     \
                       timing.c        version.c                       \
                       version_features.c              xtea.c
-
+endif
 SRC_X509_CSRCS =      certs.c         pkcs11.c        x509.c          \
```

Comment:
Do we require SRC_X509_CSRCS  for CONFIG_TC_SFS?

Reply:
No. Not required. Only aes.c need to be compiled.